custom/covid/management/commands/add_assignment_cases.py

```diff
@@ -54,13 +54,16 @@ def update_cases(self, domain, case_type, user_id):
             case_ids = accessor.get_open_case_ids_in_domain_by_type(case_type, owner_ids=[location_id])
 
         case_blocks = []
+        failed_cases = []
         skip_count = 0
         for case in accessor.iter_cases(case_ids):
             if case.get_case_property('current_status') == 'closed':
                 skip_count += 1
             elif needs_update(case):
                 new_owner_id = find_owner_id(case, accessor)
                 if new_owner_id is None:
+                    failed_cases.append(f"CaseNotFound: case:{case.case_id} no matching case for this case's "
+                                        "assigned_to_primary_checkin_case_id")
```

Comment:
I spoke with Kim that just the case_id is most important to have, but also since that information is so easily accessible I added it into the statement.